framework/src/test/java/io/github/kgress/scaffold/webdriverwrapper/WebDriverWrapperTests.java

```diff
@@ -48,4 +47,12 @@ public void testFindElementsStrongType() {
     public void testFindElementDoesntExist() {
         assertThrows(NoSuchElementException.class, () -> mockWebDriver.findElement(By.id("elementDoesNotExist")));
     }
+
+    @Test
+    public void testSetSeleniumTimeout() {
+        var newTimeout = 10L;
+        assertNotEquals(webDriverWrapper.getSeleniumObjectTimeout(), newTimeout);
```

Comment:
what is the old timeout? is it expected to be populated initially (i believe this is the casE), or null? instead of calling notEquals(), i suggest asserting the state of the default... that is assertEquals(currentVal, expectedDefaultVal) to ensure that it's 1. set, and 2. set correctly.

Reply:
Updated